content/en/events/2020/kcc2020/_index.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
We can drop 2020 from the directory name, the file path should look like this:
`content/en/events/2020/kcc/_index.md`